Cargo.toml

```diff
@@ -1,3 +1,23 @@
 [workspace]
 
-members = ["xline", "curp", "benchmark", "utils", "engine", "xlineapi", "curp-external-api", "curp-test-utils", "xline-client"]
+members = [
+  "xline",
+  "curp",
+  "benchmark",
+  "utils",
+  "engine",
+  "xlineapi",
+  "curp-external-api",
+  "curp-test-utils",
+  "xline-client",
+  "simulation",
+]
+
+# TODO: We should remove it when the madsim updates its release in the crate.io
+# TODO: We also should include the original crate's version of madsim packages
+[patch.crates-io]
```

Comment:
To enable the "patch" feature, we need to install the `cargo-patch` crate. Please update it in the quick start.

Reply:
I think patch is supported by cargo? https://doc.rust-lang.org/cargo/reference/overriding-dependencies.html